test/xmlServer.js

```diff
@@ -5,8 +5,9 @@ import serveStatic from 'serve-static'
 import path from 'path'
 
 export async function startServer () {
-    var app = express()
-    app.use(serveStatic(path.join(__dirname)))
-    const url = await listen(http.createServer(app))
-    return url
+    var app = express();
+    const server = http.createServer(app);
+    app.use(serveStatic(path.join(import.meta.dirname)))
```

Comment:
The `import.meta.dirname` was introduced in Node 20.11, so most tests fail on Node 16/18.

Personally, I'm running TerriaMap with Node 20 without issues, so I'm fine with bumping the requirements, but you seem keen on supporting Node releases way past their end of life date.

Reply:
Thanks for spotting it, I have switched to `import.meta.url` for now.